scripts/build.d/python

```diff
@@ -49,7 +49,18 @@ pushd ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src/${pkgfull} > /dev/null
   buildcmd make.log make build_all -j ${NP}
   buildcmd install.log make install
 
-  curl https://bootstrap.pypa.io/get-pip.py | ${PREFIX}/bin/python3
+  # check python version
+  PY_BIN=${PREFIX}/bin/python3
+  CURL_URL=https://bootstrap.pypa.io/get-pip.py
+
+  IFS='.' read -ra VER <<< ${VERSION}
+
+  if [[ ${VER[0]} == "v2" ]]; then
+    PY_BIN=${PREFIX}/bin/python
+    CURL_URL=https://bootstrap.pypa.io/pip/2.7/get-pip.py
```

Comment:
Wow, I didn't know it!

Reply:
This trick is neat!

FYI: if I remember correctly, pip changed a lot recently. If some tricks work for earlier pip, it may not work for the most recent pip.